pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md

```diff
@@ -0,0 +1,91 @@
+
+
+1. Set the `VERSION` environment variable to the version of Kommander you want to install, for example:
+
+    ```bash
+    export VERSION=v2.1.1
+    ```
+
+1. Set the `TARGET_NAMESPACE` to the workspace (or project) namespace in which the catalog repository needs to be created.
+
+    ```bash
+    export TARGET_NAMESPACE=<WORKSPACE_OR_PROJECT_NAMESPACE>
+    ```
+
+1. Create environment variables that contain the Gitea credentials:
+
+    ```bash
+    GITEA_USERNAME=$(kubectl get secrets -nkommander-flux kommander-git-credentials -oyaml -o go-template="{{.data.username | base64decode }}")
+    GITEA_PASSWORD=$(kubectl get secrets -nkommander-flux kommander-git-credentials -oyaml -o go-template="{{.data.password | base64decode }}")
+    ```
+
+1. Using the above username and password, follow the steps below to login to the gitea UI and create a new repository in the `kommander` organization.   
+    
+    ```bash
+    GITEA_URL=$(kubectl -n kommander get ingress gitea -o go-template='https://{{ (index .status.loadBalancer.ingress 0).hostname }}:443/dkp/kommander/git')
+    ```
+
+1. Login to the Gitea dashboard at `GITEA_URL` with the `GITEA_USERNAME` and `GITEA_PASSWORD` credentials. Create a new repository named `dkp-catalog-applications` (or any other name you prefer) under `kommander` organization.
+
+1. Clone the newly created repository on your local machine:
+
+    ```bash
+    git clone -c http.sslVerify=false https://$GITEA_USERNAME:$GITEA_PASSWORD@$GITEA_URL/kommander/dkp-catalog-applications
+    ```
+
+1. Download and extract the catalog repository bundle on to your local machine from the download portal and extract the contents into `dkp-catalog-applications` cloned earlier:
+
+    ```bash
+    curl -fsSL https://github.com/mesosphere/dkp-catalog-applications/archive/refs/tags/$VERSION.tar.gz | tar zxf - --strip-components=1 -C dkp-catalog-applications
+    ```
+
+1. Navigate into the `dkp-catalog-applications` directory and push the changes:
+
+    ```bash
+    cd dkp-catalog-applications
+    git add .
+    git commit -m "feat: initialize dkp-catalog-applications for airgapped"
+    git push --set-upstream origin master
+    ```
+
+1. Run the following command to create a secret containing Gitea credentials in `TARGET_NAMESPACE`:
+
+    ```bash
+    kubectl create secret generic -n${TARGET_NAMESPACE} ${TARGET_NAMESPACE} --type opaque \
+        --from-literal=caFile="$(kubectl get secrets -nkommander-flux kommander-git-credentials -o template='{{ .data.caFile | base64decode }}')" \
+        --from-literal=username="$(kubectl get secrets -nkommander-flux kommander-git-credentials -o template='{{ .data.username | base64decode }}')" \
+        --from-literal=password="$(kubectl get secrets -nkommander-flux kommander-git-credentials -o template='{{ .data.password | base64decode }}')"
```

Comment:
we MUST copy username/password because we disable anonymous clones in our gitea installation https://github.com/mesosphere/kommander-applications/blob/main/services/gitea/4.1.1/defaults/cm.yaml#L33 explicitly cc @mikolajb @makkes feel free to suggest any optimizations we can do in this guide.

Reply:
This will grant any user on the attached cluster admin access to the management cluster's Git repo which is not something we should encourage. Users should rather create users in Gitea that only have access to that specific repository.

Reply:
actually the secret needs to be created on mgmt cluster as well as managed clusters. mgmt cluster secret is used by gitrepository_apps controller to create apps. managed cluster secret is used by services deployed on them. i ll update the note saying that this secret needs to be created on both.

Reply:
Updated the instructions to create a new gitea user instead of copying kommander user credentials.